targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_MCU_K64F/device/TOOLCHAIN_IAR/MK64FN1M0xxx12.icf

```diff
@@ -61,6 +57,13 @@ if (!isdefinedsymbol(MBED_APP_SIZE)) {
     define symbol MBED_APP_SIZE = 0x100000;
 }
 
+if (!isdefinedsymbol(MBED_BOOT_STACK_SIZE)) {
+    define symbol MBED_BOOT_STACK_SIZE = 0x400;
+}
+
+define symbol __stack_size__=MBED_BOOT_STACK_SIZE;
```

Comment:
@theotherjimmy does the name **MBED_BOOT_STACK_SIZE** seems reasonable to you? Alternatively I could set this to the target define that is set - **MBED_CONF_TARGET_BOOT_STACK_SIZE**. Let me know what you think.